specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/AdvancedThreatProtection/DeleteAdvancedThreatProtectionSettings_example.json

```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "api-version": "2017-08-01-preview",
+    "resourceUri": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/SampleRG/providers/Microsoft.Storage/storageAccounts/samplestorageaccount",
+    "settingName": "current"
+  },
+  "responses": {
+    "200": {
```

Comment:
missing 204 example (copy from the JIT delete)

you also have a test that fails on that

Reply:
I'm removing the DELETE operation and stays only with GET and PUT.
When no settings are exists, GET will return "isEnalbed : false" value.
Using PUT, we'll be able to turn the settings ON or OFF.